### PR TITLE
fix: incorrect logos styles

### LINKED ISSUE #89: The logos has incorrect style
**Describe the bug**

We have regression issues due to https://github.com/verdaccio/ui/pull/80

Verdaccio_-_webpack

The issue seems we forgot to remove some quotes https://github.com/verdaccio/ui/pull/80/files#diff-2780c4d0882ad10abae71aa33af34c7bR147

It affects loading logo and header logo.

Issues

Screen Shot 2019-07-07 at 1 02 02 PM

Screen Shot 2019-07-07 at 1 01 14 PM

### PR DESCRIPTION
**Pull request by sergiohgz**
**Type: fix**

The following has been addressed in the PR:

*  There is a related issue? #89 
*  Unit or Functional tests are included in the PR: Updated snapshots related

**Description:** This fixes the logo styles following next:
1. Remove unnecessary quotes that surrounds value in Emotion css templated string. This happens only when using templated string, with object syntax quotes are mandatory to transform value in a string.
2. Revert a change made in #47, where `md` prop where removed from Logo component. After revert and set a bigger size when `md` (the same value before that PR), logo is centered and have correct styles.

### CODE REVIEW

**Comment by priscilawebdev** [comment]
@sergiohgz Thank you!!

File: src/components/Loading/Loading.tsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ import { Wrapper, Badge } from './styles';
 const Loading: React.FC = () => (
   <Wrapper>
     <Badge>
-      <Logo />
+      <Logo md={true} />
```
Comment (griffithtp) [inline src/components/Loading/Loading.tsx]: what do you think about a `size` prop taking enums instead which will allow multiple size in future for the Logo component?

Comment (juanpicado) [reply src/components/Loading/Loading.tsx]: I'd rather remove the parameter, I think we will never have `90px` there. We even suggest use `40px` in the documentation.

Comment (juanpicado) [reply src/components/Loading/Loading.tsx]: Aside of that, it works perfect now.

Comment (sergiohgz) [reply src/components/Loading/Loading.tsx]: Yeah, that approach was there before #47, so the first step is revert to that moment and later improve with an enum. I've just updated, take a look again @juanpicado @griffithtp

**Pull request merged**